pkgs/development/ocaml-modules/cudf/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, fetchurl, stdenv, ocaml, ocamlbuild, findlib, ocaml_extlib, glib, perl, pkg-config, stdlib-shims, ounit }:
+
+stdenv.mkDerivation {
+  pname = "ocaml${ocaml.version}-cudf";
+  version = "0.9";
+
+  src = fetchurl {
+    url = "https://gforge.inria.fr/frs/download.php/36602/cudf-0.9.tar.gz";
+    sha256 = "sha256-mTLk2V3OI1sUNIYv84nM3reiirf0AuozG5ZzLCmn4Rw=";
+  };
+
+  buildFlags = [
+    "all"
+    "opt"
+  ];
+  buildInputs = [
+    findlib
+    glib
+    ocaml
+    ocaml_extlib
+    ocamlbuild
+    perl
+    pkg-config
+    stdlib-shims
+  ];
+
+  checkTarget = [
+    "all"
+    "test"
+  ];
+  checkInputs = [
+    ounit
+  ];
+  doCheck = true;
+
+  installPhase = ''
+    runHook preInstallPhase
+    mkdir -p $out/lib/ocaml/${ocaml.version}/site-lib/cudf
+    make install BINDIR=$out
+    runHook postInstallPhase
+  '';
```

Comment:
`BINDIR` is defined on [Makefile](https://scm.gforge.inria.fr/anonscm/gitweb?p=cudf/cudf.git;a=blob;f=Makefile;h=edf24c44cabd9de3f501cf4b96fca8f1c61efeb6;hb=HEAD). 

It outputs binaries to root of `$out`, and libraries to `$out/lib/ocaml/${ocaml.version}/site-lib/cudf`. If I don't create the folder, it errors as:
 > ocamlfind: Bad configuration: Cannot mkdir /nix/store/3i6z3lwr9nacz37nrmi3xj4r3598wbq3-ocaml4.12.0-cudf-0.9/lib/ocaml/4.12.0/site-lib/cudf because a path component does not exist or is not a directory

Regarding the binaries (cudf-check, cudf-parse-822) I'm now moving them to `$out/bin`.

I understand you don't want me to overwrite the installPhase, but I could not make it work without doing so.